CHANGELOG.md

```diff
@@ -8,7 +8,18 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 ### Added
+
+## [0.2.3] - 2022-10-05
+### Added
 * Support for MySQL storage backend (#210) (by @MidasLamb)
+* Support for type restrictions in authorization models (#223)
```

Comment:
I don't like the wording here, because it may be confused that we are actually supporting this.

Can we phrase it something like:
```
Allow indicating user type to restrict in authorization models (#223) - Note: Type restriction is not enforced yet.
```

Reply:
How about
> Allow specification of type restrictions in authorization models (#223) - Note: Type restriction is not enforced yet, this just allows storing them.